assets/javascripts/discourse/controllers/admin-plugins-explorer.js

```diff
@@ -175,10 +175,8 @@ export default Controller.extend({
   },
 
   actions: {
-    dummy() {},
-
-    expandSchema() {
-      this.set("hideSchema", false);
+    updateHideSchema(value) {
+      this.set("hideSchema", value);
```

Comment:
Is `hideSchema` used anywhere still in the controller/its template?

Reply:
It is used the in `admin-plugins-explorer` template!